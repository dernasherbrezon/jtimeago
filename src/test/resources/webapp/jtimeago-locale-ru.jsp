<%@page contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" trimDirectiveWhitespaces="true" %>
<%@ taglib prefix="ta" uri="https://github.com/dernasherbrezon/jtimeago" %>
<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
<%
	java.util.Date comment = new java.util.Date(1534320716000L);
	pageContext.setAttribute("comment", comment);
%>
<fmt:setLocale value="ru"/>
<ta:formatDate value="${comment}" pattern="dd MMMM yyyy HH:mm" />