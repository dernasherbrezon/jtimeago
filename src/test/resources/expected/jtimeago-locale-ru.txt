15 авг 2018 08:11